### PR TITLE
Update remove-users.md

### PR DESCRIPTION
**Pull request by McCoyBot**
Per @salarson.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: SharePoint/SharePointOnline/remove-users.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ This article describes how to remove deleted users so they no longer appear in S
  
     When a user or guest browses to a SharePoint site, their user information is cached in the UserInfo list. When the user or guest is deleted, their related UserInfo information is not removed. Their profile still appears, which may cause confusion when people view the people picker.
 
-- **Scenario 2: A mismatched PUID.**
+- **Scenario 2: Site User ID Mismatch .**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/remove-users.md:35]:
```suggestion
- **Scenario 2: Site User ID Mismatch.**
```

File: SharePoint/SharePointOnline/remove-users.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ This article describes how to remove deleted users so they no longer appear in S
...
+        This issue most frequently occurs when a user is deleted and the account is then re-created with the same user name. The account in the Microsoft 365 admin center or Active Directory (in directory synchronization scenarios) is deleted and re-created with the same user principal name (UPN). The new account is created by using a different ID value. When the user tries to access a site collection or their OneDrive, the user has an incorrect ID. A second scenario involves directory synchronization with an Active Directory organizational unit (OU). If users have already signed in to SharePoint, and then are moved to a different OU and resynced with SharePoint, they may experience this problem.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/remove-users.md:37]:
```suggestion
     This issue most frequently occurs when a user is deleted and the account is then re-created with the same user name. The account in the Microsoft 365 admin center or Active Directory (in directory synchronization scenarios) is deleted and re-created with the same user principal name (UPN). The new account is created by using a different ID value. When the user tries to access a site collection or their OneDrive, the user has an incorrect ID. A second scenario involves directory synchronization with an Active Directory organizational unit (OU). If users have already signed in to SharePoint, and then are moved to a different OU and resynced with SharePoint, they may experience this problem.
```

**Comment by JohanFreelancer9** [comment]
Copy looks good but please remove extra white spaces.

**Pull request merged**